app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope ->{order created_at: :desc}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :use, presence: true 
```
sự khác biệt của chúng là gì?

Reply:
`validates :user_id, presence: true`: Yêu cầu phải có trường dữ liệu id của người dùng.
`validates :user, presence: true`: Yêu cầu phải có record người dùng.
Theo cơ chế belongs_to thì micropost sẽ tham chiếu user nên cả 2 trường hợp đều đúng.
Còn theo tutorial thì create sẽ sử dụng người dùng hiện tại để tạo nên không có trường hợp không xác thực được user_id theo em validate ở đây để tường minh.